neurodamus/core/configuration.py

```diff
@@ -626,7 +627,8 @@ def _global_parameters(config: _SimConfig, run_conf):
     h.celsius = config.celsius
     h.set_v_init(config.v_init)
     h.tstop = config.tstop
-    h.dt = run_conf.get("Dt", 0.025)
+    config.default_neuron_dt = h.dt
```

Comment:
Just to be sure that this change is equivalent: is `h.dt` already 0.025 by default?

Reply:
At the moment yes :+1: